standards/SPIFFE_Workload_API.md

```diff
@@ -203,7 +194,7 @@ The `X509SVIDRequest` request message is currently empty and is a placeholder fo
 
 The `X509SVIDResponse` response consists of a mandatory `svids` field, which MUST contain one or more `X509SVID` messages (one for each identity granted to the client). The `crl` and `federated_bundles` fields are optional. 
 
-All fields in the `X509SVID` message are mandatory.
+All fields in the `X509SVID` message are mandatory, with the exception of the `hint` field. When the `hint` field is set, its value MUST be unique amongst the set of returned SVIDs in the `X509SVIDResponse` message.
```

Comment:
```suggestion
All fields in the `X509SVID` message are mandatory, with the exception of the `hint` field. When the `hint` field is set (i.e. non-empty), its value MUST be unique amongst the set of returned SVIDs in any given `X509SVIDResponse` message.
```

Trying to clarify that the uniqueness applies to a specific instance of `X509SVIDResponse`.